### ISSUE TITLE
Overseerr Not Allowing User to Navigate Away From Application

### ISSUE THREAD

**Issue by yammes08** [sct/overseerr#3146]
### Description

When using Overseerr, and typing either a web address or a search term in your browser’s search bar and hitting enter/tapping go, you are returned to the Overseerr interface and not to your target destination. 

### Version

796b18fe1cb85e9ebdffedc9ebec223f908cbe30

### Steps to Reproduce

1. Navigate to Overseerr either by http://ip:port or by using https://overseerr.mydomain.com
2. Type “google.com” in your browser’s search bar
3. Press go or hit enter
4. Observe that you are returned to the Overseerr interface and not google.com

### Platform

desktop

### Device

iPhone 13 Pro Max, Windows 11

### Operating System

iOS 16.1, Windows 11

### Browser

iOS Safari, Windows Firefox

### Code of Conduct

- [x] I agree to follow Overseerr's Code of Conduct

**Comment by emilnymann** [comment]
I'm not able to reproduce.

**Comment by yammes08** [comment]
I've gathered some more information on this. Firstly though, apologies, it doesn't look as though this behaviour exhibits under Windows. iOS still exhibits this behaviour, though.

I am running Overseerr in Unraid/Docker using a custom docker network.

Here are the scenarios I've tested:

Enable Proxy Support Disabled > access Overseer via http://[ip]:port > behaves normally
Enable Proxy Support Disabled > access Overseerr via external url > anomalous behaviour
Enable Proxy Support Enabled > access Overseer via http://[ip]:port > anomalous behaviour (I suspect this is expected)
Enable Proxy Support Enabled > access Overseerr via external url > anomalous behaviour

I'm using NginxProxyManager as my reverse proxy.
